react/src/components/toggle/SprkToggle.js

```diff
@@ -9,10 +9,15 @@ import 'focus-visible';
 class SprkToggle extends Component {
   constructor(props) {
     super(props);
+    // TODO: Remove isDefaultOpen in issue #1296
     const { isDefaultOpen } = this.props;
+    let { isOpen } = this.props;
+    if (isDefaultOpen !== undefined && isOpen === undefined) {
+      isOpen = isDefaultOpen;
+    }
     this.state = {
-      isOpen: isDefaultOpen || false,
-      height: isDefaultOpen ? 'auto' : 0,
+      isOpen: isOpen || false,
+      height: isOpen ? 'auto' : 0,
```

Comment:
i would assign the new isOpen from the old isDefaultOpen, then you dont have to have logic to check both:
```suggestion
    const {
      isDefaultOpen,
      isOpen = isDefaultOpen,
    } = this.props;

    this.state = {
      isOpen: isOpen || false,
      height: isOpen ? 'auto' : 0,
```